### ISSUE TITLE
Neuroglancer Viewer has many accessibility issues

### ISSUE THREAD

**Issue by jingdid** [google/neuroglancer#659]
Upon running an Axe Devtools scan of our neuroglancer page, we surfaced a very large number of accessibility issues that seem inbuilt to the Neuroglancer library. 

For example, class `.neuroglancer-side-panel-titlebar.neuroglancer-layer-side-panel-title` gets assigned `background-color: #393;` while the text on it in `.neuroglancer-layer-side-panel-name` gets assigned `color: white;` and this is not a high enough contrast ratio, resulting in this message: Element has insufficient color contrast of 3.65 (foreground color: #ffffff, background color: #339933, font size: 9.0pt (12px), font weight: bold). Expected contrast ratio of 4.5:1

Most other issues surfaced were either due to "Form elements must have labels" or "<svg> elements with an img role must have an alternative text."

**Comment by jbms** [comment]
Potentially we could tweak the colors a bit.  I would indeed consider issues like that in-scope.  As far as I am aware Neuroglancer, being a visualization tool, is not usable anyway by users that rely on a screen reader, for example, and therefore accessibility issues related to screen reader usage are irrelevant.

**Comment by jingdid** [comment]
Tweaking the colors would be very helpful! Not catering to screen reader usage seems pretty reasonable.
